projects/kit/components/input-phone-international/test/international-input-phone.component.spec.ts

```diff
@@ -205,40 +205,41 @@ describe('InputPhoneInternational', () => {
             expect(testComponent.control.value).toEqual('+380123456789');
         });
 
-        it('should not update value on paste if there is no country code in list', () => {
+        it('should set country code on paste event 8(863)', () => {
```

Comment:
Why was it necessary to rewrite the tests, and not add new ones?

Reply:
sounds like a `feat` ))

Reply:
ok I'l rename it :)